mappings/net/minecraft/datafixer/fix/BitStorageAlignFix.mapping

```diff
@@ -1,3 +1,47 @@
 CLASS net/minecraft/class_5196 net/minecraft/datafixer/fix/BitStorageAlignFix
+	FIELD field_29863 ELEMENT_BIT_SHIFT I
+	FIELD field_29864 UNUSED_CHUNK_WIDTH I
+	FIELD field_29865 UNUSED_CHUNK_DEPTH I
```

Comment:
Don't prefix names with `UNUSED_`. Also "height" might work better than "depth".

Reply:
Width = X
Depth = Z

I definitely did not want to use "height" because worlds aren't traditionally 16 blocks tall.

Reply:
They might have been used to compute other constants. That might be why they appear as "unused" when you look up their literal values in decompiled code.
```suggestion
	FIELD field_29864 CHUNK_WIDTH I
	FIELD field_29865 CHUNK_LENGTH I
```
Just random guesses, who knows which one is length or which one is width

Reply:
Honestly I prefer `X_SIZE` and `Z_SIZE`, it's less confusing.

Reply:
@liach They are unused because they are unused. The fields are private and their values do not occur anywhere in the class.